Dockerfile

```diff
@@ -46,6 +46,8 @@ ENV INVENTREE_BACKGROUND_WORKERS="4"
 ENV INVENTREE_WEB_ADDR=0.0.0.0
 ENV INVENTREE_WEB_PORT=8000
 
+ENV VIRTUAL_ENV=/usr/local
```

Comment:
Does this mean we now use a venv again in the docker image?

Reply:
No, this just sets the venv variable to the python path

Reply:
Ok